### DISCUSSION TITLE
thanhle.blog/my-2022

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#77] [Ideas]
# thanhle.blog/my-2022

Lại tới dịp review lại năm cũ rồi

https://thanhle.blog/blog/my-2022

**Comment by thanhlmm** [comment]
Trong bài viết có lẽ bạn sẽ thấy mình mention tới $ khá nhiều. Thật ra thì mình cũng khá ngại khi nói ra những cái đó, vì ngoài kia còn nhiều người làm điều đó tốt hơn mình.
Ý mình muốn deliver ở đây là hãy làm mọi việc tốt tới mức người khác trả tiền cho bạn vì điều đó

**Reply by phungvansyhb** [reply]
$ chỉ là minh chứng cho những gì a nói thôi mà :)) . E tình cờ đọc được bài chia sẻ của a về làm freelancer trên upwork xong e follow a luôn . Lối viết của a cuốn zl 💁‍♂️💁‍♂️ Đọc xong e cũng có động lực làm cái blog giống của a quá :))

**Comment by darkpiv** [comment]
Tuyệt vời e :D Tiền trước giờ là 1 thước đo kết quả mà. Người ta paid mình well khi mình deliver được gì đó hoy.

**Comment by nguyenhoaikhang37** [comment]
Năm cũ qua đi, năm mới đã tới. Chúc anh Hậu bầu trời sức khỏe, biển cả tình thương, đại dương tình bạn, sự nghiệp sáng ngời, gia đình thịnh vượng nhé.🎉🎉 Năm 2022 một trong những bước tiến trong sự nghiệp là việc biết được đến blog của anh, tuy không có cơ hội đi với anh lâu dài nhưng em rất cảm ơn vì những kiến thức và góc nhìn mà anh đã mang lại. Cũng chúc blog mình năm mới ra những bài blog thật xịn^^

**Reply by thanhlmm** [reply]
Cảm ơn em nhiều. Chúc em sức khỏe và thuận lợi trong công việc nha 😆
